### ISSUE TITLE
Quest can't start

### ISSUE THREAD

**Issue by hktomiki** [PikaMug/Quests#1652]
Hi,

I have bug, that i created one quest, where is must to click one npc and then go to 1-3 diggerent locations. Then i created similar one, but player can't start the quest, because it is already done.
¨
Spigot version: API version 1.16.5-R0.1-SNAPSHOT

**Comment by PikaMug** [comment]
Thanks for the report. Please include your Quests version and example quest, as mentioned in the Report a Bug template.

**Comment by hktomiki** [comment]
Quests 4.0.1 quest is: citizens2 npc To start quest. Quest is start, and go To over. But when I create New similar quest, plugin say that There is No quests. When I type /quests join test plugin is say that I have maked the quest already.

1. Npc start
2. Go To location 1
3. Go To location 2
4. Quest is ready

That quest I tried make 2 timed. I did removed the npc and quest, but its still say that, I have maked the quest.
